### ISSUE TITLE
Swift Package Manager Support?

### ISSUE THREAD

**Issue by tristanl-slalom** [CA-Application-Performance-Management/CAMobileAppAnalytics#3]
I'm working at a company that is aiming to move away from Cocoapods and toward Swift Package Manager. 

Does CAMobileAppAnalytics plan on supporting SPM in the future?

Thank you for your help, great library!

**Comment by ArunaYarra** [comment]
Yes, @trisco2001 we are planning to support SPM but there is no ETA yet.

Thank you.

**Comment by ArunaYarra** [comment]
We are already supporting SPM. Please go through the README.
